### PR TITLE
feat: Add support for nightmode switch

### PR DESCRIPTION
**Pull request by jonathanrobichaud4**

### CODE REVIEW

File: custom_components/ha_blueair/switch.py [hunk 1]
```diff
@@ -62,3 +63,22 @@ async def async_turn_on(self, **kwargs):
...
         self.async_write_ha_state()
+        
+class BlueairNightModeSwitchEntity(BlueairEntity, SwitchEntity):
+    _attr_device_class = SwitchDeviceClass.SWITCH
+
+    def __init__(self, device):
+        super().__init__("Night Mode", device)
+
+    @property
+    def is_on(self) -> int | None:
```
Comment (dahlb) [inline custom_components/ha_blueair/switch.py:74]: looks like this should be -> bool | None from ToggleEntity

**Comment by dahlb** [comment]
three lint errors
custom_components/ha_blueair/blueair_aws_data_update_coordinator.py:149:1: W293 [*] Blank line contains whitespace
custom_components/ha_blueair/blueair_data_update_coordinator.py:90:1: W293 [*] Blank line contains whitespace
custom_components/ha_blueair/switch.py:66:1: W293 [*] Blank line contains whitespace

**Comment by dahlb** [comment]
once the api pr is merged please update 
requirements.txt and manifest.json with newly released version for blueair-api

File: custom_components/ha_blueair/switch.py (hunk 1 above)
Comment (jonathanrobichaud4) [reply custom_components/ha_blueair/switch.py:74]: Sounds good I just used the same code for the child lock switch which is an int but that makes more sense.

**Comment by jonathanrobichaud4** [comment]
> once the api pr is merged please update requirements.txt and manifest.json with newly released version for blueair-api

Will do a separate commit for that, I'm just submitting what I have now for your review!

**Comment by dahlb** [comment]
Needs an upstream merge to resolve a conflict in requirements from ruff being updated by dependabot

**Comment by jonathanrobichaud4** [comment]
Ok fixed!

**Pull request merged**